var/spack/repos/builtin/packages/libsonata/package.py

```diff
@@ -32,6 +32,9 @@ class Libsonata(CMakePackage):
     depends_on('highfive+mpi', when='+mpi')
     depends_on('highfive~mpi', when='~mpi')
     depends_on('mpi', when='+mpi')
+    depends_on('py-pybind11', when='@0.1.3:')
```

Comment:
Isn't this required only when python bindings are required? (in case there is `+python` variant)

Reply:
I cannot see python variant inside this package. Can this variant come from outerspace (an other file)?

Reply:
There is no variant. It's a separate package, `py-libsonata`.

Reply:
This dependency should then be dropped.

Reply:
Got it. Patch in progress